src/chronometer.js

```diff
@@ -1,33 +1,49 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
   }
 
   start(printTimeCallback) {
-    // ... your code goes here
+    this.intervalId = setInterval(() => {
+      this.currentTime++;
+      if (printTimeCallback) {
+        printTimeCallback(this.currentTime);
+      }
+    }, 1000);
   }
 
   getMinutes() {
-    // ... your code goes here
+    return Math.floor(this.currentTime / 60);
   }
 
   getSeconds() {
-    // ... your code goes here
+    return Math.floor(this.currentTime % 60);
   }
 
   computeTwoDigitNumber(value) {
-    // ... your code goes here
+    let toString = value.toString();
+    let nums = toString.split("");
+    return nums.length === 1 ? "0" + toString : toString;
   }
 
   stop() {
-    // ... your code goes here
+    clearInterval(this.intervalId);
   }
 
   reset() {
-    // ... your code goes here
+    return (this.currentTime = 0);
   }
 
   split() {
-    // ... your code goes here
+    let minTimeStamp = this.getMinutes(this.currentTime);
```

Comment:
this split method works, but it is not well written, because it doesn't utilize this.computeTwoDigitNumber nicely. 

have a look at Antonio's solution:
```js
let min=this.computeTwoDigitNumber(this.getMinutes());
let sec=this.computeTwoDigitNumber(this.getSeconds());
return `${min}:${sec}`;
```

Reply:
now for this to look as short as it does, you would have to rewrite this.getMinutes and this.getSeconds as well. They don't need an argument of this.currentTime, because they can just access anything from the object instance by this  anyways.